.github/workflows/builds.yml

```diff
@@ -171,16 +171,34 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [macos-11, macos-12]
+       include:
+         - os: macos-11
+           qt-version: 5.15
+           qt-major: 5
+           min-macOS-version: 10.9
+         - os: macos-12
+           qt-version: 6.6
+           qt-major: 6
+           min-macOS-version: 11
+           qt-modules: qt5compat
+
 
     env:
       VERBOSE: 1
       TERM: xterm-256color
 
     steps:
+      - name: Install Qt
+        uses: jurplel/install-qt-action@v3
+        with:
+          dir: /Users/runner
```

Comment:
I think I'd rather use a GH Actions context variable here.

Reply:
In this case there is no reason we don't use it again.
In the future i plan to unify the steps and use this action for all at that point i will use a variable

Reply:
I think I'd be in favour of improving these things at the beginning.

Reply:
Un needed so remvoed